### DISCUSSION TITLE
How to cancel single scan/template

### DISCUSSION THREAD

**Discussion by Drjacky** [projectdiscovery/nuclei#4507] [Q&A] [answered]
In the middle of scan, one of the scans(templates) stuck. How can I cancel that one and let the scanner continue?
```bash
.
.
.
[some-PIIs] [http] [medium] https://example.com/ []
[INF] Using Interactsh Server: oast.fun STUCK HERE
```

**Comment by olearycrew** [comment, marked as answer]
That isn't possilbe @Drjacky - you'd have to split it up into multiple scans
